01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,18 @@
+def fizzbuzz(number1=3, number2=5, max_count=20)
+    max_count.times do |i|
+        i += 1
+        common_multiple = number1*number2
```

Comment:
演算子の前後にはスペースを入れておくと読みやすくなります ✨ `%` を使っている部分なども合わせて修正してみてください。

```suggestion
        common_multiple = number1 * number2
```